int64 class_id
geometry_msgs/Point pose